### DISCUSSION TITLE
Formula Help documentation

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#495] [Q&A]
I am trying to get my head around the formula property but the help documentation is very limited and I can't seem to get the Luxon information to work.

Some working examples in the help doc would be useful.

I thought putting ${Now()} would put now in the formula box but I get nothing.

Maybe an error message would be useful.

**Comment by RafaelGB** [comment]
If you enable the developer logs on global settings of the plugin you can see the error message per row inside the console

**Reply by RafaelGB** [reply]
Also I am agree with the documentation, but first it would be necessary to achieve a more mature state of the functionality.  I'll spend some time on it once I'm done with the nested metadata thing.

**Reply by DannyHatcher-Git** [reply]
> If you enable the developer logs on global settings of the plugin you can see the error message per row inside the console

I will have to check that out!

I am trying to work out math and it just leaves me with nothing sometimes. My assumption is that I need to turn the row into a number rather than string.

The difference between dates/times is what I am troubleshooting first

**Reply by RafaelGB** [reply]
Luxon's library is quite advanced and date management is a complex topic in js. It has its difficulty curve. We will iterate to make it as less steep as possible
